specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2023-09-01-preview/nodetype.json

```diff
@@ -0,0 +1,1962 @@
+          "description": "Specifies the DSCP configuration to apply to the network interface."
+        },
+        "ipConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpConfiguration"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "Specifies the IP configurations of the network interface."
+        }
+      },
+      "description": "Specifies the settings for a network interface to attach to the node type."
+    },
+    "DiskType": {
+      "type": "string",
+      "description": "Managed data disk type. IOPS and throughput are given by the disk size, to see more information go to https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types.\n",
+      "enum": [
+        "Standard_LRS",
+        "StandardSSD_LRS",
+        "Premium_LRS"
+      ],
+      "default": "StandardSSD_LRS",
+      "x-ms-enum": {
+        "name": "DiskType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Standard_LRS",
+            "description": "Standard HDD locally redundant storage. Best for backup, non-critical, and infrequent access."
+          },
+          {
+            "value": "StandardSSD_LRS",
+            "description": "Standard SSD locally redundant storage. Best for web servers, lightly used enterprise applications and dev/test."
+          },
+          {
+            "value": "Premium_LRS",
+            "description": "Premium SSD locally redundant storage. Best for production and performance sensitive workloads."
+          }
+        ]
+      }
+    },
+    "EvictionPolicyType": {
+      "type": "string",
+      "description": "Specifies the eviction policy for virtual machines in a SPOT node type.",
+      "enum": [
+        "Delete",
+        "Deallocate"
+      ],
+      "x-ms-enum": {
+        "name": "EvictionPolicyType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Delete",
+            "description": "Eviction policy will be Delete for SPOT vms."
+          },
+          {
+            "value": "Deallocate",
+            "description": "Eviction policy will be Deallocate for SPOT vms."
+          }
+        ]
+      }
+    },
+    "FrontendConfiguration": {
+      "type": "object",
+      "properties": {
+        "ipAddressType": {
+          "$ref": "#/definitions/IPAddressType",
+          "description": "The IP address type of this frontend configuration. If omitted the default value is IPv4."
+        },
+        "loadBalancerBackendAddressPoolId": {
+          "type": "string",
+          "description": "The resource Id of the Load Balancer backend address pool that the VM instances of the node type are associated with. The format of the resource Id is '/subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.Network/loadBalancers/<loadBalancerName>/backendAddressPools/<backendAddressPoolName>'."
```

Comment:
format the property as arm-id 
[https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details)